tests/Unit/Domain/Test_DomainTestHelpers.cpp

```diff
@@ -77,3 +84,38 @@ SPECTRE_TEST_CASE("Unit.Domain.TestHelpers.VolumeCornerIterator",
   test_vci_2d();
   test_vci_3d();
 }
+
+namespace {
+
+template <size_t SpatialDim, typename SpatialFrame>
+void test_spatial_basis_vectors(const DataVector& used_for_size) noexcept {
+  const auto directions = Direction<SpatialDim>::all_directions();
+
+  for (size_t i = 0; i < directions.size(); ++i) {
```

Comment:
`for(const auto& direction : Direction<SpatialDim>::all_directions()){
...
}`
you wont need line 92 nor line 95